lib/iris/tests/integration/test_aggregated_cube.py

```diff
@@ -37,15 +35,16 @@ def test_agg_by_aux_coord(self):
         cube = iris.load_cube(problem_test_file)
 
         # Test aggregating by aux coord, notably the `forecast_period` aux
-        # coord on `cube`, whose `_points` attribute is of type
-        # :class:`biggus.Array`. This test then ensures that
-        # aggregating using `points` instead is successful.
+        # coord on `cube`, whose `_points` attribute is a lazy array.
+        # This test then ensures that aggregating using `points` instead is
+        # successful.
 
-        # First confirm we've got a `biggus.Array`.
+        # First confirm we've got a lazy array.
         # NB. This checks the merge process in `load_cube()` hasn't
         # triggered the load of the coordinate's data.
         forecast_period_coord = cube.coord('forecast_period')
-        self.assertIsInstance(forecast_period_coord._points, biggus.Array)
+        self.assertTrue(forecast_period_coord._points.all(),
```

Comment:
Something gone wrong here ?
I think this means "unless all points are zero, raise [the function] ```iris._lazy_data.is_lazy_data``` as an error message :  It does not _call_ the function.
I'm also not sure what .all() does : does this produce a computed equality check, or is a dask deferred object ?

Equivalent to old code I ***think*** should be just
```
self.assertTrue(iris._lazy_data.is_lazy_data(forecast_period_coord._points))
```

**Stop press:** I checked out dask ```.all()``, and it does return a _deferred_ object.
This probably means that we should be using "array.all()" more in place of "np.all(array)", as the array function generalises to work properly on either kind of object.